### PR TITLE
ut(ticdc): add table actor ut

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: ref #3881

### What is changed and how it works?
add table actor ut

### Check List 

Tests 

 - Unit test

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

**Comment by ben1009** [comment]
```
[](https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/4354/pipeline/#step-49-log-453)[](https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/4354/pipeline/#step-49-log-454)[](https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/4354/pipeline/#step-49-log-455)[](https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/4354/pipeline/#step-49-log-456)[](https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/4354/pipeline/#step-49-log-457)[2022-03-07T15:58:06.608Z]     client_test.go:1343: 

[2022-03-07T15:58:06.608Z]         	Error Trace:	client_test.go:1343

[2022-03-07T15:58:06.608Z]         	Error:      	Not equal: 

[2022-03-07T15:58:06.608Z]         	            	expected: 1

[2022-03-07T15:58:06.608Z]         	            	actual  : 0

[2022-03-07T15:58:06.608Z]         	Test:       	TestStreamSendWithError
```

File: cdc/processor/pipeline/actor_node.go [hunk 1]
```diff
@@ -54,9 +55,15 @@ func (n *ActorNode) TryRun(ctx context.Context) error {
...
-		} else {
+		// node is blocked
+		if !ok {
+			return nil
+		}
+
+		n.messageStash = nil
+		processedCount++
+		// processed too many messages, not block other tables
+		if processedCount >= defaultOutputChannelSize {
```
Comment (overvenus) [inline cdc/processor/pipeline/actor_node.go:66]: Could you elaborate the scenario you want to address?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: cdc/processor/pipeline/actor_node.go (hunk 1 above)
Comment (sdojjy) [reply cdc/processor/pipeline/actor_node.go]: done

**Review by overvenus — APPROVED** [review]

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**